lib/genres.dart

```diff
@@ -0,0 +1,39 @@
+import 'package:flutter/material.dart';
+import 'movie.dart';
+
+class Genres extends StatelessWidget {
+  final Movie _mov;
+
+  const Genres(this._mov, {super.key});
+
+  static const genresWordPadding = EdgeInsets.only(left: 20);
+  static const genresTextStyle = TextStyle(
+    color: Colors.white,
+    fontWeight: FontWeight.bold,
+    fontSize: 22,
+  );
+
+  List<Widget> _createWidgets() {
+    List<Widget> widgetedGenres = [];
+    _mov.genres.forEach((element) {
+      widgetedGenres.add(
```

Comment:
Minor improvement: you could use `.map(...)` method to achieve the same!
`widgetedGenres = _mov.genres.map(...)`

Reply:
Thanks for the advice and the explanation!